tests/hpc/mpi_imb.pm

```diff
@@ -0,0 +1,114 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Basic MPI integration test using IMB.
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base qw(hpcbase hpc::utils), -signatures;
+use testapi;
+use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use lockapi;
+use utils;
+use registration;
+use version_utils 'is_sle';
+use Utils::Logging 'export_logs';
+use hpc::formatter;
```

Comment:
What is the use of `hpc::formatter` in bellow code? I don't see it.

Reply:
removed